### DISCUSSION TITLE
Any way to support TLS authentication and simple authorization without Common Name

### DISCUSSION THREAD

**Discussion by prateekkohli21** [strimzi/strimzi-kafka-operator#7821] [Q&A]
I need to use mTLS authentication and simple authorization using Strimzi kafka.
As per the documentation to support simple authorization we need to add the KafkaUser name as CN in client certificate.

But if I use external certificates, is there a way to authorize with simple authorization without using client certificate's CN?
Any other way to add username in client which can be verified by the server except certificate's CN.

**Comment by scholzj** [comment]
If you want to use the User Operator, you have to follow the user operator rules and have the certificates with the `CN=<KafkaUserName>`. You can also disable the USer Operator and manage things your self directly using the Kafka Admin APis any way you want.
